tests/unit/changeDir.ts

```diff
@@ -0,0 +1,22 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import changeDir from 'src/changeDir';
+import { stub, SinonStub } from 'sinon';
+
+let changeDirStub: SinonStub;
+
+registerSuite({
+	name: 'changeDir',
+	'setup'() {
+		changeDirStub = stub(process, 'chdir');
```

Comment:
Another nice trick is to write a module which exports the `process` global, and then mock that module import.